tests/training/metrics/evalb_bracketing_scorer_test.py

```diff
@@ -0,0 +1,28 @@
+# pylint: disable=no-self-use,invalid-name,protected-access
+import os
+from nltk import Tree
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.training.metrics import EvalbBracketingScorer
+
+
+class EvalbBracketingScorerTest(AllenNlpTestCase):
```

Comment:
This test is pretty budget but the EVALB software is already pretty well tested, along with NLTK's Tree object. Let me know if there are any specific ones that you'd like to see.

Reply:
Having one test with non-perfect scores would be nice, just as a simple sanity check.